JPF/bad_buildings.sql

```diff
@@ -56,6 +80,16 @@ drop table if exists boro_lookup;
 ################
 create temporary table boro_lookup select boro, boroid from hpd_buildings group by 1,2;
```

Comment:
I figured this out and thought I had deleted this comment but hadn't, sorry. I think the indexes are clear enough here, but a little confusing in the longer statements.